src/outagesim/backup_reliability.jl

```diff
@@ -505,77 +767,217 @@ function backup_reliability_inputs(d::Dict, p::REoptInputs; r::Dict = Dict())::D
     end
     
     #If gen capacity is 0 then base on diesel_kw
-    #If num_gens is zero then either set to 1 or base on ceiling(diesel_kw / gen_capacity)
-    gen_capacity = get(r, "gen_capacity", 0)
-    num_gens = get(r, "num_gens", 1)
-
-    if gen_capacity < 0.1
-        if num_gens <= 1
-            gen_capacity = diesel_kw
-            num_gens = 1
+    #If num_generators is zero then either set to 1 or base on ceiling(diesel_kw / generator_size_kw)
```

Comment:
If the user provides a reopt run and a generator_size_kw to override that but not num_generators, should num_generators be set to 1 or reopt results gen size divided by generator_size_kw?
@Bill-Becker

Reply:
I think the former, but just wanted to check